src/components/Carousel/index.tsx

```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
 import styled, { css } from '../../utils/styled';
-import { KeyCodes } from '../../utils/keyCodes';
-import { remCalc } from '../../utils/remCalc';
+import { KeyCodes, remCalc } from '../../utils';
```

Comment:
Not 100% sure about this (there might have been a reason for having explicit module imports instead of taking the `index`, maybe one of the utils refers to components which would make it cyclic?). Have you checked that this works?

If yes, then fine with me.

Reply:
Yes I'm 100% sure it works cause I've checked it on local machine (put my case into runnable docs) and code has passed all the tests.